client/app/telecom/telephony/alias/special/fees/telecom-telephony-alias-special-fees.html

```diff
@@ -0,0 +1,124 @@
+<section class="telephony-group-alias-special-fees">
+    <header>
+        <oui-back-button data-ui-sref="telecom.telephony.alias.special"
```

Comment:
Related to the [component API](https://master.ui-kit.ovh/#!/oui-angular/back-button), why not using `on-click`?

Reply:
`onclick` shall bring a `$state.go` dependency no ?